platform/broadcom/saibcm-modules/debian/opennsl-modules.init

```diff
@@ -32,14 +32,42 @@ function create_devices()
 #                                 level logs
 function load_kernel_modules()
 {
-    modprobe linux-kernel-bde dmasize=32M maxpayload=128 debug=4 dma_debug=1
+    onie_platform="$(cat /host/machine.conf | grep onie_platform | cut -d= -f2)"
```

Comment:
there is aboot_platform as well. you can just source it.

Reply:
Done